docs/development.md

```diff
@@ -100,18 +106,18 @@ To debug the code generator:
 
 ### Built in commands
 
-There are a number of custom rush commands to help with development. See the `.scripts` folder for more.
+There are a number of custom pnpm commands to help with development. See the `.scripts` folder for more. Add the `-r` switch for the script to apply to every package.
```

Comment:
Add the -r switch when executing the script from a directory other than the root. This will execute the script for all sub-directories of the current directory. (something like that)